src/transformers/models/opt/modeling_opt.py

```diff
@@ -60,6 +63,10 @@
 _SEQ_CLASS_EXPECTED_OUTPUT = "'LABEL_0'"
 
 
+def _shape(tensor: torch.Tensor, seq_len: int, bsz: int, num_heads: int, head_dim: int) -> torch.Tensor:
+    return tensor.view(bsz, seq_len, num_heads, head_dim).transpose(1, 2).contiguous()
```

Comment:
I agree in general this is better as a function, but this breaks the pattern with other models so it's better to just leave as-is